api/src/main/resources/CES_Drug_List-4.csv

```diff
@@ -155,62 +211,89 @@
 "e33a7015-5398-4ac8-93e2-b6a5a315ed87",,"CES: Shampoo vin.quassia para piojos","CIEL:82983",
 "c12ca1aa-5ce0-40f3-ba2c-6398180b0908",,"CES: Sildenafil tabletas **especial**","CIEL:83902",
 "950a9c0c-0c3e-4c52-ac85-118e0542fdbe",,"CES: Simvastatina 10mg - tabletas","CIEL:83936",
 "ac3fdeaf-c736-4162-a16c-cc3b78c2ae8d",,"CES: Sulfadiazina de plata crema 1g","PIH:938",
+"db1ac6c1-0ec6-4262-aeaf-8ec6c1871e45",,"CES: Sulfato de magnesio inyectable 10% - 1 ampolla **especial**","CIEL:79247",
 "7c46d158-bd95-4644-95c4-920dd227a31b",,"CES: Sulfazalacina 500mg tabletas **especial**","CIEL:84480",
+"cdf54234-42a7-4616-b915-03d597693c1e",,"CES: Sumatriptán 100 mg tabletas","CIEL:84522",
+"88bf2fa8-fc2c-4640-805c-93b087658aef",,"CES: Tacrolimus 1% crema **especial**","CIEL:84646",
 "82a31c5c-f7fe-49dc-8bc0-185c41d9a64a",,"CES: Tamoxifeno 20 mg - 14 tabletas **especial**","PIH:7664",
 "4e3454aa-3861-4653-83cd-b1f9ba0b0ddd",,"CES: Tamsulosina 0.4mg - 20 capsulas","CIEL:84670",
 "ddd1e000-5fbd-4fe5-be51-df2bddef465a",,"CES: Telmisartan 40mg - 14 tabletas","CIEL:104843",
 "60c5aed0-b5cb-4dc3-a2fb-bb94ad7f1d66",,"CES: Telmisartan/hidroclorotiazida 80mg/12.5mg - tabletas","CIEL:104843",
 "bc4009f5-c22a-404a-a62a-93249d27a558",,"CES: Terazosina 2mg 20 tabletas","CIEL:84823",
+"4d009d36-b1b8-44af-875b-aff3fab25da9",,"CES: Tetraciclina - tableta - 500 mg","CIEL:84893",
 "0f9fe499-3cd8-4c48-ab7d-43a8fd62cc8e",,"CES: Tiamazol 5mg - 20 tabletas **especial**","CIEL:79691",
 "8fe01874-8a7f-457d-9505-3a5fdd7cefcb",,"CES: Timolol + brimonidina + dorzolamida gotas oftalmicas **especial**","CIEL:104546",
+"b1d19f31-f38e-493b-bd6d-64c8bcf001e3",,"CES: Tobramicina/dexametasona uguento oftalmico","PIH:12258",
+"83f5e6ac-9eda-4299-aace-a7f854a13923",,"CES: Tolterodina tabletas **especial**","PIH:12259",
 "eedd4433-24a6-47fd-ae54-fdac03f6e369",,"CES: Topiramato 100mg - 20 tabletas","CIEL:85235",
+"e049d0b0-9053-44db-ab5f-ebb47696e3fe",,"CES: Topiramato 25mg - 20 tabletas","CIEL:85235",
 "7ec0852e-0beb-49cf-b325-7fff31be54f4",,"CES: Tramadol - capsulas - 100 mg","CIEL:85276",
 "3d735fd4-0c78-4c15-a166-979d366495d5",,"CES: Tramadol inyectable 50mg/ml 5 ampollas de 2ml","CIEL:85276",
 "60c5a970-281c-4d53-b361-80b1fe5efbe1",,"CES: Trimetroprima/sulfametoxazol 160mg/800mg - 14 tabletas","CIEL:105281",
+"e159f0db-4a34-4964-ae8d-4233fb56648d",,"CES: Trimetroprima/sulfametoxazol 80mg/400mg - 20 tabletas","CIEL:105281",
 "e2ab1d6c-fe74-4d73-bf0f-80f01db5f864",,"CES: Trimetroprima/sulfametoxazol suspencion 80 mg/400 mg/100 ml","CIEL:105281",
 "4638b608-a7dd-4ad8-87e4-e6d7c32a6ac6",,"CES: Valproato de magnesio 200mg - 40 tabletas","CIEL:79251",
-"76604add-8992-4d8d-a17b-42b1fc36c580",,"CES: Vaselina - 60g","CIEL:81861",
-"cf0c064c-1b39-4914-9006-037d12684660",,"CES: Venlafaxina 75mg - 20 tabletas","CIEL:86155",
-"fc738280-504b-4a0d-8259-7454a6162db9",,"CES: Vitaminas angels","PIH:461",
-"e864bcd1-b735-48d2-a55a-74fc56bf5741",,"CES: Zinc 0.38 g capsulas","PIH:4064",
```

Comment:
Did we decide that we are okay with these actually *not* being removed?  

Since they aren't being removed behind the scenes, would it be better *not* to remove them here, to maintain consistency between what is in this file and what is in the DB?

Reply:
Oh yeah, good catch. I'll put those back.